include/mbedtls/constant_time.h

```diff
@@ -23,20 +23,22 @@
 
 #include <stddef.h>
 
-
 /** Constant-time buffer comparison without branches.
  *
  * This is equivalent to the standard memcmp function, but is likely to be
- * compiled to code using bitwise operation rather than a branch.
+ * compiled to code using bitwise operation rather than a branch, such that
```

Comment:
grr, missed this before
```suggestion
 * compiled to code using bitwise operations rather than a branch, such that
```